sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.students.controller.getScholasticStandingSummary.e2e-spec.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentScholasticStanding,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import {
+  BEARER_AUTH_TYPE,
+  FakeStudentUsersTypes,
+  createTestingAppModule,
+  getStudentByFakeStudentUserType,
+  getStudentToken,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+import { Student } from "@sims/sims-db";
+import { saveFakeSFASIndividual } from "@sims/test-utils/factories/sfas-individuals";
+
+describe("StudentScholasticStandingsStudentsController(e2e)-getScholasticStandingSummary.", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let student: Student;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    student = await getStudentByFakeStudentUserType(
```

Comment:
The shared student is not supposed to have its data or its related data changed since it can cause side effects to other E2E tests in the future.
Can we create a fake user for these tests?
If the `saveFakeStudent` is used the SIN will also be available and there will no need to execute the below code in the 47.
https://github.com/bcgov/SIMS/blob/f0109ee2e44ac0b870e5c1a9ee408b2ff286ad8f/sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.students.controller.getScholasticStandingSummary.e2e-spec.ts#L45-L48

Reply:
I agree. Please follow getStudentProfile test as sample. 
![image](https://github.com/bcgov/SIMS/assets/54600590/ab8cbd10-4204-4357-bd1c-129f6f5c4f7b)